### PR TITLE
refactor: remove unused robot param

### LINKED ISSUE #40: Refactor: Remove extra robot param
https://github.com/gr2m/wip-bot/blob/6d67f35f4168338a82b91885e16459e6aaf3df8c/lib/handle-pull-request-change.js#L3

Not in use anywhere.

### PR DESCRIPTION
**Pull request by anshumanv**
Fixes #40

### CODE REVIEW

File: lib/handle-pull-request-change.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
 module.exports = handlePullRequestChange
 
-async function handlePullRequestChange (robot, context) {
+async function handlePullRequestChange (context) {
```
Comment (gr2m) [inline lib/handle-pull-request-change.js]: removing it here is not enough, the robot argument is still passed at 

https://github.com/gr2m/wip-bot/blob/6d67f35f4168338a82b91885e16459e6aaf3df8c/index.js#L10

We'll have to remove the pre-binding of the robot argument there (remove `.bind(null, robot)`)

Comment (anshumanv) [reply lib/handle-pull-request-change.js]: Ah, dang. How did I forget :sweat_smile:

Comment (anshumanv) [reply lib/handle-pull-request-change.js]: Done the required changes, squashed the commits. :+1:

**Comment by gr2m** [comment]
Oops something’s broken https://zeit.co/gr2m/wip-bot/zrbzofgipy/logs I’ll look into it

**Comment by anshumanv** [comment]
Can we do something like - 
```
module.exports =  (robot) => {
  const events = [
    'pull_request.opened',
    'pull_request.edited',
    'pull_request.synchronize'
  ]
  robot.on(events, async context => {
    handlePullRequestChange(context))
  }   
}
```

I'm using this in a personal bot of mine and it seems to be working this way. @gr2m

**Comment by gr2m** [comment]
it was unrelated to your change, I'm not yet sure what  it is. I just wanted to let you know in case you have seen that it went down. #43 fixed the problem, I'll investigate another time. If you like, try to make the wip-bot run locally and see if you can reproduce it. I guess it's something related to a probot dependency introduced in 5.0.0

**Pull request merged**